airbyte-integrations/connectors/source-file/metadata.yaml

```diff
@@ -5,7 +5,7 @@ data:
   connectorSubtype: file
   connectorType: source
   definitionId: 778daa7c-feaf-4db6-96f3-70fd645acc77
-  dockerImageTag: 0.3.6
+  dockerImageTag: 0.3.7
```

Comment:
We also need to update the cloud pinned version

https://github.com/airbytehq/airbyte/pull/26131/files#diff-580b4a25c9d6e8ce3172870d4a6eb1064426fc4c557c69b90361902bc66d39dfL17